cmd/ci-clean/main.go

```diff
@@ -179,6 +179,7 @@ func deleteKeys(ctx context.Context, metalClient *packet.Client, keys metal.SSHK
 }
 
 func deleteEMLBPools(ctx context.Context, emlbClient *emlb.EMLB, pools *lbaas.LoadBalancerPoolCollection) error {
+	ctx = context.WithValue(ctx, lbaas.ContextOAuth2, emlbClient.TokenExchanger)
```

Comment:
I don't see how this adds does anything different than what is already being done within the Delete methods. Those Delete methods will override this tokenexchanger with the same tokenexchanger.

https://github.com/kubernetes-sigs/cluster-api-provider-packet/pull/776/files#diff-da50f13120ab2041469b613ebd889e387b7e8df3f038824d730d728048f4f6baR473

Reply:
Good catch. That was my first attempt at resolving the permission issue and turned out it was the wrong path.

Reply:
Did this merge fix anything?

My point wasn't about this one particular line, so it matches the entire PR. converting the private variables to public variables should make 0 difference from what I see in this PR.